server/src/main/java/io/spine/server/event/EventStore.java

```diff
@@ -131,17 +147,75 @@ public void run() {
                     responseObserver.onNext(event);
                 }
                 responseObserver.onCompleted();
-                logCatchUpComplete(responseObserver);
+                logReadingComplete(responseObserver);
             }
         });
     }
 
+    @VisibleForTesting
+    Executor getStreamExecutor() {
+        return streamExecutor;
+    }
+
+    /**
+     * Closes the underlying storage.
+     *
+     * @throws IOException if the attempt to close the storage throws an exception
+     */
+    @Override
+    public void close() throws Exception {
+        storage.close();
+    }
+
+    private void logStored(Event request) {
+        if (logger == null) {
+            return;
+        }
+        if (logger.isTraceEnabled()) {
+            logger.trace("Stored: {}", TextFormat.shortDebugString(request));
+        }
+    }
+
+    private void logReadingStart(EventStreamQuery request, StreamObserver<Event> responseObserver) {
+        if (logger == null) {
+            return;
+        }
+
+        if (logger.isInfoEnabled()) {
+            final String requestData = TextFormat.shortDebugString(request);
+            logger.info("Creating stream on request: {} for observer: {}",
+                        requestData,
+                        responseObserver);
+        }
+    }
+
+    /*
+     * Logging methods
```

Comment:
Let's format the section separator according to [our guidelines](https://github.com/SpineEventEngine/core-java/wiki/Java-Code-Style#comment-separators-for-sections-of-the-class).